app/src/main/java/com/ucm/informatica/spread/Activities/MainTabActivity.java

```diff
@@ -134,6 +139,14 @@ public void hideLoading() {
 
     }
 
+    public void saveDataPoster(String posterJson){
```

Comment:
every load/save data with outside ddbb should be done from main presenter and distribute to each fragment, before it was made in Map, Home and Historical...x3